pages/answers/[section]/index.js

```diff
@@ -0,0 +1,55 @@
+import sectionsData from "data/sectionsData";
+
+const sectionsPage = ({ data, sectionsData }) => {
```

Comment:
lets not use catch all api routes as its hard to determine which slug we are targeting